lesson-1/scrips.js

```diff
@@ -0,0 +1,10 @@
+let user = {
+    name: 'Josep',
+    age: 25,
+    hobbies: ['bassketball' ,'massage' ,'work'],
+    sayHello () {
+        console.log(`Hello, My name is ${this.name}. I am ${this.age} years old. I have ${this.hobbies.length} hobbies: ${this.hobbies}`)
+    }   
+}
+
+user.sayHello();
```

Comment:
leave an empty string in the end of every file
```suggestion
user.sayHello();

```